lib/mhv/oh_facilities_helper/service.rb

```diff
@@ -23,13 +27,21 @@ def user_facility_ready_for_info_alert?
       private
 
       def pretransitioned_oh_facilities
-        @pretransitioned_oh_facilities ||=
-          Settings.mhv.oh_facility_checks.pretransitioned_oh_facilities.split(',').map(&:strip)
+        @pretransitioned_oh_facilities ||= parse_facility_setting(
+          Settings.mhv.oh_facility_checks.pretransitioned_oh_facilities
+        )
       end
 
       def facilities_ready_for_info_alert
-        @facilities_ready_for_info_alert ||=
-          Settings.mhv.oh_facility_checks.facilities_ready_for_info_alert.split(',').map(&:strip)
+        @facilities_ready_for_info_alert ||= parse_facility_setting(
+          Settings.mhv.oh_facility_checks.facilities_ready_for_info_alert
+        )
+      end
+
+      def parse_facility_setting(value)
+        return [] if value.blank? || value == false || value == 0
```

Comment:
It may do more than you want, but you might want to consider using `ActiveModel::Type::Boolean.new.cast(value)` as it handles all of the edge cases for what should be considered truthy. It would consider strings like "false" and "0" to be false though (along with the non string versions of those), so it is pretty heavy handed

So something like 
`return [] unless ActiveModel::Type::Boolean.new.cast(value)`